### ISSUE TITLE
Merge RNA-seq evidence (ECO:0000295) and RNA sequencing assay evidence (ECO:0001821)? 

### ISSUE THREAD

**Issue by pgaudet** [evidenceontology/evidenceontology#143]
Hello, 

These 2 terms seem equivalent: 
RNA-seq evidence (ECO:0000295)
RNA sequencing assay evidence (ECO:0001821)

I suggest merging them.

Thanks, Pascale

**Comment by mchibucos** [comment]
Hi Pascale,
Becky just checked, and it appears that the term ECO:0000295 has some child terms and is older. ECO:0001821 is newer, and to our knowledge is not used. Do you use either of these terms? I'm thinking we should obsolete 1821, as it should not have ever been created...
Marcus

**Comment by pgaudet** [comment]
hi Marcus, 

I don't mind either way. The safest would be a merge (give ECO:0001821 as a secondary ID of ECO:0000295) but obsoleting also works for me. 

Thanks ! 

Pascale

**Comment by rctauber** [comment]
Hi @pgaudet - I just pushed a commit to deprecate ECO:0001821 (RNA sequencing assay evidence), as well as it's 'used in' children (ECO:0005664 and ECO:0005665). We chose to deprecate instead of merging because 'RNA sequencing assay evidence' should not have been created, and to our knowledge these terms are not yet being used for active annotation.

RNA-seq evidence (ECO:0000295) is the suggested term to use, and I created the 'used in' children for this term:
RNA-seq evidence used in manual assertion (ECO:0006068)
RNA-seq evidence used in automatic assertion (ECO:0006069)

These changes will be live next Friday (7/7). See also @mchibucos on #144. Thanks!
